Reneiski_Homework1/ReneiskiSelenium11/PageObjects/BasePage.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OpenQA.Selenium;
+using ReneiskiSelenium11.Common.Drivers;
+using ReneiskiSelenium11.Common.WebElements;
+
+namespace ReneiskiSelenium11.PageObjects
+{
+    public class BasePage : WebDriverFactory
+    {
+        private MyWebElement elementsByXPath = new MyWebElement(By.XPath("//*[@class=\"card-body\"]//*[contains(text(), 'Elements')]"));
+
+        public void GoToElemntsPage() => elementsByXPath.Click();
+
+        public string TextResult(string value)
```

Comment:
do you really need this method? you can just find elements by XPath in the test class and call .Text()

Reply:
Fixed